lib/form526_backup_submission/service.rb

```diff
@@ -5,90 +5,20 @@
 require 'common/client/errors'
 require 'common/exceptions/forbidden'
 require 'common/exceptions/schema_validation_errors'
-require 'form526_backup_submission/configuration'
+require 'benefits_intake_service/service'
+require_relative 'configuration'
```

Comment:
I think require_relative is usually discouraged 🤔 .... I am wondering why this is being done as a require relative instead of just a normal require? There might be a valid reason, I just dont know what it is.

Reply:
Hmm didn't realize, it looks clear this way to me but I have no preference